### DISCUSSION TITLE
I can't add or update any meetings on the plugin

### DISCUSSION THREAD

**Discussion by jackieoh1** [code4recovery/12-step-meeting-list#694] [Q&A] [answered]
Hi all - I can't update any meetings on the plugin right now.  I also can't switch a meeting from a draft to published - the publish button either stays grayed out, or in the case of draft-to-published, the publish button sometimes appears to be active but is inop.  Can I please get some help with this?  Thank you!  (I also commented in #683)

Best -

Jodie Schanhals
Northern Michigan Central Office
info@nmcentraloffice.org
206.310.1833

**Comment by kiyote33** [comment]
Hi @jackieoh1, 
I'd be happy to help if I can get a bit more info. Please advise **what theme you are using** and I will see if I can replicate your issue and suggest a solution.  Please note that we do have an update patch coming soon for a related geo-coding issue which may also resolve your problem, but in the meantime I may be able to suggest something if I can duplicate what's going on when you are editing/creating meeting records.

**Comment by jackieoh1** [comment]
Sure. I'm using the 2014 theme, and switched over a couple months ago to using this [tsml_react] on a page (not sure if that matters or not). Thank you.

**Reply by kiyote33** [reply]
Well your issue doesn't appear to be theme related as I can't repro it using the 2014 theme. Maybe if I try loading your meetings it would help me replicate it.  Can you provide your CSV or the data source link your using?

**Reply by kiyote33** [reply]
I loaded 129 records from "Grand Traverse County" via the meetings feed from your site.  Still not seeing your problem.  Can you suggest a meeting that you are having problems editing?

**Comment by jackieoh1** [comment]
Again - the publish button is either not "engaging" when an edit is made, or in the case of trying to publish the draft meeting (The Foothills Group), I can make some edits to make the Publish button to appear active, but clicking on it does not publish the meeting.

Yesterday, I needed to edit the Long Lake Group to get rid of the zoom link - the publish button never activated.
I also tried to create a meeting - the publish button never activated.

I hope this helps, and please let me know if you need more clarification.

Thank you!
Jodie

On Feb 15, 2022, at 7:18 PM, Wayne Parker ***@***.***> wrote:


I loaded 129 records from "Grand Traverse County" via the meetings feed from your site. Still not seeing your problem. Can you suggest a meeting that you are having problems editing?

**Reply by kiyote33** [reply]
Hi @jackieoh1, 
I'm a bit stumped with this issue.  I've loaded your theme and records into my test environment, yet have no problem updating the Long Lake Group or creating a new meeting record.  I'm wondering if one of your installed plugins might be clashing with our plugin.  Could you disable them one at a time and test to see if there is a culprit there? Is this a new website?  Have you had the ability to update records in the past? If so, have you tried rolling back to a previous version of our plugin to see if the 12 Step Meeting List edits the way you expect?

**Comment by brianw-area24** [comment, marked as answer]
I'm curious what option you have selected for "Address Geocoding Method" in the Imports & Settings page? If you're using the Beta API Gateway, does it make a difference if you change this to Legacy?

**Reply by kiyote33** [reply]
Thx Brian, I've duplicated that scenario in my sandbox which is configured to mimic the https://nmcentraloffice.org/ website environment (wp version, tsml version, 2014 theme). Still not able to repro the issue with Legacy or Beta.  Hoping to get a listing of plugins loaded to see if that helps to reproduce the issue that Jodie is experiencing. I've tested using the Edge, Opera, Mozilla, and Chrome browsers.

**Reply by jackieoh1** [reply]
Brian - 
You nailed it!  I changed to legacy (I think I had changed to the Beta API Gateway recently, but don't quite remember).  After changing it, the map shows up again when editing meetings, and I'm able to create, edit, and publish from a draft.

Kiyote33 - I am using chrome (on a Macbook Air)... I'll get you the plugin info soon.
